polyfills/Event/polyfill.js

```diff
@@ -17,6 +17,11 @@
 		textinput: 1
 	};
 
+	// This polyfill depends on availability of `document` so will not run in a worker
+	// However, we asssume there are no browsers with worker support that lack proper
+	// support for `Event` within the worker
+	if (!document || !window) return;
```

Comment:
This will throw a `ReferenceError` where `document` or `window` are not defined.

Instead, this could be written as `typeof document === 'undefined' || typeof window === 'undefined'`.